### PR TITLE
2 bug fixes.

### PR DESCRIPTION
**Pull request by NamiUni**

### CODE REVIEW

File: src/main/java/com/ordwen/odailyquests/quests/player/progression/checkers/AbstractEntityChecker.java [hunk 1]
```diff
@@ -41,9 +41,8 @@ public void setPlayerQuestProgression(Player player, EntityType entityType, Stri
 
                 if (abstractQuest instanceof EntityQuest quest) {
 
-                    if (quest.getEntityTypes() == null) isRequiredEntity = true;
+                    if (quest.getEntityTypes() == null || quest.getEntityTypes().isEmpty()) isRequiredEntity = true;
```
Comment (Ordwen) [inline src/main/java/com/ordwen/odailyquests/quests/player/progression/checkers/AbstractEntityChecker.java:44]: The list is never created if the quest section does not contain the `required_entity` part... But it's smart to add this if someone leaves the part empty :)

File: src/main/java/com/ordwen/odailyquests/quests/player/progression/storage/sql/LoadProgressionSQL.java [hunk 2]
```diff
@@ -84,7 +84,15 @@ public void loadProgression(String playerName, HashMap<String, PlayerQuests> act
 
                     PluginLogger.info(ChatColor.GOLD + playerName + ChatColor.YELLOW + "'s quests have been loaded.");
 
-                    final String msg = QuestsMessages.QUESTS_IN_PROGRESS.toString();
+                    if (achievedQuests == playerQuests.getPlayerQuests().size()) {
+
```
Comment (Ordwen) [inline src/main/java/com/ordwen/odailyquests/quests/player/progression/storage/sql/LoadProgressionSQL.java:88]: You forgot to remove this condition, right? :')

**Comment by Ordwen** [comment]
Wow, comments directly on commits is ugly, I'll detail here instead 

For all the places where you replaced `required_entity` with `entity_type`, those were not mistakes. Since 2.0 `entity_type` has actually become `required_entity`, because it makes more sense with `required_item`, and it was a common mistake. If it doesn't work, there must be an error somewhere else 🤔 

For the EntityChecker, the list is never created if the quest section does not contain the `required_entity` part... But it's smart to add this if someone leaves the part empty 

For the message when loading quests, that's good too, I had done the same thing for YAML, but I had forgotten for SQL... But you forgot to remove a condition, right? 😂 

Thanks again, I appreciate it, especially with the little time I have with studies

**Comment by NamiUni** [comment]
Sorry, I should have checked the commit log properly. It seems that when I sent a pull request for a bug fix the other day, the bug was caused by using the bug-fixed version as is on the server.
If you had separated the branches for version 1.0 and 2.0, there might not have been any confusion...w I don't mean to complain, don't worry! Please separate the branches in the next major update!

I'm going to close this pull request and send a new pull request for SQL message modification. Thank you!

**Pull request closed without merging**